daemon/api_snapshots_test.go

```diff
@@ -380,3 +382,42 @@ func (s *snapshotSuite) TestExportSnapshotsBadRequestOnError(c *check.C) {
 	c.Check(rsp.Result, check.DeepEquals, &daemon.ErrorResult{Message: `cannot export 1: boom`})
 	c.Check(snapshotExportCalled, check.Equals, 1)
 }
+
+func (s *snapshotSuite) TestImportSnapshot(c *check.C) {
+	// mock snapshot export file
+	data := []byte("mocked snapshot import data file!")
```

Comment:
Nitpick, the comment seems redundant, maybe the contents should just say "mocked snapshot **export** data file!"